src/demos/demos.css

```diff
@@ -1,26 +1,33 @@
 body {
   margin: 0;
-  background-color: #dfdfdf;
+  /* background-color: #dfdfdf; */
+  background-color: var(--calcite-app-background-content);
+}
 
+main {
+  margin: 0;
+  padding: 0;
 }
+
 .example-container {
-  padding: 0 24px 24px;
-}
-@media (min-width: 1025px) {
-  .example-container {
-    width: 900px;
-    max-height: 80vh;
-    margin: 12px auto;
-    background-color: var(--calcite-app-background-content);
-    border: 1px solid var(--calcite-app-foreground-hover);
-    box-shadow: 4px 4px 2px var(--calcite-app-foreground-hover);
-    resize: horizontal;
-    overflow-x: scroll;
-  }
+  color: var(--calcite-app-foreground-subtle);
+  max-width: 1024px;
+  min-width: 480px;
+  width: 70vw;
+  padding: var(--calcite-app-cap-spacing-half) var(--calcite-app-side-spacing-double) var(--calcite-app-cap-spacing-double);
+  margin: 0 auto var(--calcite-app-cap-spacing-double);
+  background-color: var(--calcite-app-background-content);
+  box-shadow: var(--calcite-app-shadow-1);
 }
 
-h1, h2, h3, h4, h5 {
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: 'Courier New', Courier, monospace;
   font-weight: 300;
+  font-size: 14px;
```

Comment:
What do you think about increasing the font size and uppercasing the h1 so that the component name sticks out a bit better?

Reply:
On a second review, all of the headings are kind of getting lost on the page. I like the courier font but I think this font size may be a bit too small for effectively dividing the sections of the demo up.

Reply:
This was in response to @driskull comment that the headers get mixed in with the components. Their size, color, and font-family are all contrasting to the main content, i.e. the components.

Does that make sense?

Reply:
Actually, the h1 was too little. Upped its size and weight.